diff --git a/src/HMRClient.ts b/src/HMRClient.ts
index 994006c0876f5c009a9bbf4b557e8657355bcadc..8e5335113df2f2e8eb408f5696a364751dd1b0ae 100644
--- a/src/HMRClient.ts
+++ b/src/HMRClient.ts
@@ -8,7 +8,7 @@
  * Based on this but with web support:
  * https://github.com/facebook/react-native/blob/086714b02b0fb838dee5a66c5bcefe73b53cf3df/Libraries/Utilities/HMRClient.js
  */
-import MetroHMRClient from 'metro-runtime/src/modules/HMRClient';
+const MetroHMRClient = require('metro-runtime/src/modules/HMRClient');
 import prettyFormat, { plugins } from 'pretty-format';
 
 import LoadingView from './LoadingView';
